.gitignore

```diff
@@ -16,8 +16,9 @@ coverage
 _config-deploy.yml
 _config-deploy.ymle
 js/app.js
-_cf_s3_invalidator.yml
 .vagrant
 ubuntu-xenial-16.04-cloudimg-console.log
 .jekyll-metadata
-
+_cf_s3_invalidator.yml
```

Comment:
does the ordering here matter? it looks like this line was removed and then re-added later?

Reply:
This was a blank line being removed and added the AWS yaml file to .gitignore to keep it from being committed.